tests/testthat/test_posterior_predict.R

```diff
@@ -253,13 +253,21 @@ test_that("compatible with stan_lmer with offset", {
 })
 
 context("posterior_predict (stan_betareg)")
-test_that("compatible betareg", {
+test_that("compatible with stan_betareg with z", {
   data("GasolineYield", package = "betareg")
-  fit <- SW(stan_betareg(yield ~ temp, data = GasolineYield, 
-                     iter = ITER*5, chains = CHAINS, seed = SEED, refresh = REFRESH))
+  fit <- SW(stan_betareg(yield ~ batch + temp | temp, data = GasolineYield,
+                         iter = ITER, chains = CHAINS, seed = SEED, 
+                         refresh = REFRESH))
   check_for_error(fit)
   expect_linpred_equal(fit)
```

Comment:
@imadmali this test I added to `test_posterior_predict` for `stan_betareg` fails. I added a test for a model including a `z` term because previously there was only one without it. It's just the `expect_linpred_equal` test that fails. What does `posterior_linpred` do for `stan_betareg` models that have `z` terms?